lib/StringClient.js

```diff
@@ -12,12 +12,15 @@ var util = require('util');
 
 var HttpClient = require('./HttpClient');
 
+var nodeMajorVer = parseInt(process.versions.node.split('.'), 10);
```

Comment:
This is a constant -- can you upper case this?

Reply:
Sure.